ssh/tailssh/tailssh.go

```diff
@@ -51,6 +51,11 @@ var (
 	sshDisableSFTP       = envknob.RegisterBool("TS_SSH_DISABLE_SFTP")
 	sshDisableForwarding = envknob.RegisterBool("TS_SSH_DISABLE_FORWARDING")
 	sshDisablePTY        = envknob.RegisterBool("TS_SSH_DISABLE_PTY")
+
+	// terminalError is an empty gossh.PartialSuccessError (with no 'Next'
+	// authentication methods that may proceed), which results in the SSH
+	// server immediately disconnecting the client.
+	terminalError = &gossh.PartialSuccessError{}
```

Comment:
error variables are named `errFoo` (or `ErrFoo`) in Go, and types are named `fooError` (or `FooError`)

So this would be `errTerminal` or `errNoMoreAuthTypes` if you want to be more explicit.